sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/IntervalMetricReader.java

```diff
@@ -104,6 +111,11 @@ public void run() {
       }
     }
 
+    CompletableResultCode forceFlush() {
+      run();
+      return exportResult.get();
```

Comment:
there's a race here. `run()` could finish before the result is accessed, in which case this will return a null, which I think is a bad API. Take a look at the implementation in the BatchSpanProcessor for a clean way to deal with this race.